docs/2.8.0/docs/canton/usermanual/persistence.rst

```diff
@@ -456,6 +500,8 @@ If you are unsure how to size your connection pools,
 `this article <https://github.com/brettwooldridge/HikariCP/wiki/About-Pool-Sizing>`_
 may be a good starting point.
 
+As a rule of thumb, the number of connections should not be more then two times the number of CPUs on the database machine.
```

Comment:
Is this so? I suspect that this depends on the latency between the Canton node and the DB server. If the latency is high, I suspect that more connections may be useful. But then we're screwed anyway for latency.

Reply:
Yes, that's the TLDR of the above article.